Automattic/wp-calypso
# Domains: Update Domain Mapping checkout page

#### Changes proposed in this Pull Request
This PR implements the changes on the domain mapping checkout page layout. 
Here's a comparison on how they looked before vs. how they're looking now: 

#### Desktop

##### Before
<img width="1437" alt="before - desktop" src="https://user-images.githubusercontent.com/18705930/129769526-cb639bc4-1a05-42b3-8c3b-5265156914fa.png">

##### After
<img width="1452" alt="Screen Shot 2021-08-19 at 22 16 35" src="https://user-images.githubusercontent.com/18705930/130163865-c36785f1-aafe-4696-8620-7463a7ef746b.png">


#### Mobile
##### Before
<img width="546" alt="before - mobile" src="https://user-images.githubusercontent.com/18705930/129769644-05a8a46d-880e-47dd-b2e4-135da58cf5d2.png">

##### After
<img width="383" alt="Screen Shot 2021-08-19 at 22 16 55" src="https://user-images.githubusercontent.com/18705930/130163874-f4195e1f-339a-46b3-8fac-81c028d762c2.png">


#### Testing instructions
 - Add a domain mapping to the cart (you will probably need to change the back end code to allow you to do so, otherwise this can only be done if your user is DWPO exempt);
 - Checkout and confirm that the new "thank you" page looks like this. 

_(If you already have a domain mapped and you have the receipt ID, you can bypass the checkout by going to `/checkout/thank-you/:your-site/:receipt-id` 😉)_
